packages/babel-traverse/src/scope/index.ts

```diff
@@ -5,7 +5,6 @@ import type { TraverseOptions } from "../index";
 import Binding from "./binding";
 import globals from "globals";
 import {
-  FOR_INIT_KEYS,
```

Comment:
nice refactor here!

Reply:
The `FOR_INIT_KEYS` is only used here. After this PR we may even remove it in Babel 8 if nobody else is using.